packages/pds-ember/app/styles/pds/components/code-block/_config.scss

```diff
@@ -0,0 +1,24 @@
+$_module: "PDS.Components.CodeBlock";
+  /* [debug] CONFIG: #{$_module} */
+@use "../../theme";
+
+@mixin layout {
+  /* [debug] #{$_module}@layout */
+  // TBD...
+}
+
+@mixin appearance {
+  /* [debug] #{$_module}@appearance */
+  font-size: theme.$codeBlock-fontSize;
+  font-weight: theme.$codeBlock-fontWeight;
+  line-height: theme.$codeBlock-lineHeight;
```

Comment:
These styles used to be `Typography.Code()` mixin, but these are styles that will apply to an Ember component that will be built in the future, so they've been migrated to this module to serve as the start of styling for a `<Pds::CodeBlock />` component.